### DISCUSSION TITLE
Seeing protected tweets?

### DISCUSSION THREAD

**Discussion by soumyasomasundaran** [tweepy/tweepy#1832] [Questions] [answered]
Hi, Is it possible to see protected tweets of my own private account or accounts I follow using 3 Legged Oauth in tweepy V2?

**Comment by Harmon758** [comment, marked as answer]
Yes, https://tweepy.readthedocs.io/en/v4.6.0/authentication.html#legged-oauth.
If it's your own developer account, you can just [generate an access token and secret from the developer portal](https://tweepy.readthedocs.io./en/v4.6.0/authentication.html#id3).
